crates/sargon/src/system/sargon_os/transactions/sargon_os_transaction_analysis.rs

```diff
@@ -95,62 +75,20 @@ impl SargonOS {
             ChildSubintentSpecifiers::default(),
         )?;
 
-        let summary = subintent_manifest.summary()?;
-
-        if !summary.reserved_instructions.is_empty() {
-            return Err(
-                CommonError::ReservedInstructionsNotAllowedInManifest {
-                    reserved_instructions: summary
-                        .reserved_instructions
-                        .iter()
-                        .map(|i| i.to_string())
-                        .collect(),
-                },
-            );
-        }
+        let summary = subintent_manifest.validated_summary(
+            network_id,
+            false, // PreAuth transaction cannot be sent by the Host itself
```

Comment:
would be nice with a tiny newtype, so that in GH I dont see only `false` but I see `HostIsTheOrigin(false)` or similar...

Reply:
well if Rust would have hade named params...